app/views/applications/_show.html.haml

```diff
@@ -11,7 +11,11 @@
       Email
       %small (required)
     %td= @user.email
-
+  %tr
+    %td.left
+      Preferred Pronouns
+      %small (optional)
+      %small *This does not affect your application, Double Union wants to respect what pronouns you use
```

Comment:
Maybe "wants to respect appropriate pronouns"? Will have to think about how much to use "you" vs. that the pronoun is appropriate.

Or maybe "Double Union wants to respect your pronouns"